### PR TITLE
Adding helmet for securing server

### PR DESCRIPTION
**Pull request by yigitkurtcu**
Helmet paketinden önceki response headers
helmetoff
Helmet paketinden sonraki response headers
helmeton

### CODE REVIEW

File: src/lib/app.js [hunk 1]
```diff
@@ -104,8 +105,13 @@ export default class App {
       this.express.use(sentry.Handlers.requestHandler());
     }
 
-    // TODO update cors policy
-    this.express.use(Cors());
+
+    this.express.use(helmet({
+      frameguard: {
+        action: 'deny',
```
Comment (SelmanKahya) [inline src/lib/app.js]: lets also add hidePoweredBy, xssFilter

File: src/lib/app.js [hunk 2]
```diff
@@ -104,8 +105,13 @@ export default class App {
       this.express.use(sentry.Handlers.requestHandler());
     }
 
-    // TODO update cors policy
-    this.express.use(Cors());
```
Comment (SelmanKahya) [inline src/lib/app.js]: i think we still need to setup cors policy here

File: src/lib/app.js (hunk 1 above)
Comment (yigitkurtcu) [reply src/lib/app.js]: They are default helmet options. Helmet includes these modules.
https://helmetjs.github.io/

Comment (SelmanKahya) [reply src/lib/app.js]: thats right

**Pull request merged**